commands/system/help.js

```diff
@@ -1,83 +1,162 @@
-            output += `\n\`${message.settings.prefix + c.help.name}\` | ${c.help.description.length > 50 ? c.help.description.slice(0,50) +"...": c.help.description}`;
+            output += `\n\`${message.settings.prefix + c.help.name}\` | ${c.help.description.length > 80 ? `${c.help.description.slice(0, 80)}...` : c.help.description}`;
           }
           num++;
         }
       }
-    
+
       if (num) {
-        embed.setTitle("Command category help")
-          .setDescription(`A list of commands in the ${type} category.\n(Total of ${num} commands in this category)\n\nTo get help on a specific command do \`${message.settings.prefix}help <command>\`\n\n${num > 10 && pg === 1 ? `To view more commands do\` ${message.settings.prefix}help <category> 2\`` : "" }`)
+        helpembed.setTitle(`Page ${page}/${Math.ceil(num / perpage)} for ${type.toProperCase()}`)
+          .setDescription(`A list of commands in the ${type} category.\n(Total of ${num} commands in this category)\n\nTo get help on a specific command do \`${message.settings.prefix}help <command>\``)
           .addField("Commands", output);
       }
     }
 
-    if (this.client.commands.has(type) || this.client.commands.some(command => command.conf.aliases.includes(type))) {
-      const cm = this.client.commands.get(type) || this.client.commands.get(this.client.aliases.get(type));
+    if (this.client.commands.has(type) || this.client.aliases.has(type)) {
+      const cm = this.client.commands.get(type) || this.client.commands.get(this.client.aliases.get((type)));
       if (level < this.client.levelCache[cm.conf.permLevel]) return;
-      embed.setTitle(cm.help.name.toProperCase())
+      helpembed.setTitle(cm.help.name)
         .addField("Command description", cm.help.description)
         .addField("Command usage", `\`${cm.help.usage}\``)
-        .addField("Command aliases", cm.conf.aliases.length == 0 ? "None" : cm.conf.aliases.join(", ") );
+        .addField("Command aliases", cm.conf.aliases.length ? "None" : cm.conf.aliases.join(", "));
+    }
+    const msg2 = await message.channel.send(helpembed);
+    const totalpages = Math.ceil(num / perpage);
+    if (!message.guild.me.hasPermission(["MANAGE_MESSAGES"])) {
+      await message.channel.send("I don't have permission to remove reactions, please do this manually.");
     }
-    
-    if (!embed.fields[0]) return;
 
-    return message.channel.send(embed);
+    if (msg2.embeds[0].title && msg2.embeds[0].title.includes("Page") && Number(msg2.embeds[0].title.split(" ")[1].split("/")[1]) !== 1) {
+      for (const emoji of EMOJIS) await msg2.react(emoji);
+    }
+
+    const select = msg2.createReactionCollector(
+      (reaction, user) => EMOJIS.includes(reaction.emoji.name) && user.id === message.author.id,
+      { time: 300000 },
+    );
+
+    let on = false;
+    select.on("collect", async (r) => {
+      switch (r.emoji.name) {
+        case "▶":
```

Comment:
Make one variable for `Number(msg2.embeds[0].title.split(" ")[1].split("/")[0]) - 1`